app/context-map/om-workload.js

```diff
@@ -0,0 +1,6 @@
+const SourceWorkload = require('../staging/domain/om-workload')
```

Comment:
This file is not needed any more because it's replaced by `app/context-map/workload.js`